src/stream/StreamPart.js

```diff
@@ -0,0 +1,22 @@
+export default class StreamPart {
+    constructor(streamId, streamPartition) {
+        this._streamId = streamId
+        this._streamPartition = streamPartition
+    }
+
+    static fromStream({ id, partitions }) {
+        const result = []
+        for (let i = 0; i < partitions; i++) {
+            result.push(new StreamPart(id, i))
+        }
+        return result
+    }
+
+    getStreamId() {
+        return this._streamId
+    }
```

Comment:
These getters are very "Java style", seems weird, IMO we should probably just expose these as `this.streamId`, `this.streamPartition` or even `this.id` `this.partition`.

No need for getters.